flashlight/memory/managers/CachingMemoryManager.cpp

```diff
@@ -88,6 +96,25 @@ CachingMemoryManager::CachingMemoryManager(
 
 void CachingMemoryManager::initialize() {}
 
+void safeStoull(size_t& var, const char* str) {
+  if(!str)
+    return;
+  try {
+    auto value = std::stoull(str);
+    var = value;
+  } catch(const std::exception& e) {
+    // FL_LOG Warn ?
+  }
+}
+
+void CachingMemoryManager::setOption(unsigned key, const std::string& value) {
```

Comment:
As I mentioned, let's add setters on `CachingMemoryManager` directly for these parameters rather than using string option params.

Reply:
done